packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -113,6 +125,7 @@ contract SystemConfig is OwnableUpgradeable, Semver {
         batcherHash = _batcherHash;
         gasLimit = _gasLimit;
         _setUnsafeBlockSigner(_unsafeBlockSigner);
+        _setBatchInboxAddr(_batchInboxAddr);
```

Comment:
Does anyone know how the initial value is read since there is no event emitted in this case?